src/deezer/client.py

```diff
@@ -177,26 +180,29 @@ def request(
             paginate_list=paginate_list,
         )
 
-    def _get_paginated_list(self, path, **params):
-        return PaginatedList(client=self, base_path=path, **params)
+    async def _get_paginated_list(self, path, **params):
+        paginated_list: PaginatedList = PaginatedList(
+            client=self, base_path=path, **params
+        )
+        return await paginated_list.fetch()
```

Comment:
Would it make sense to set a default fetch limit?

Reply:
As i wrote above [the limit of tracks in playlists on deezer is 2000](https://support.deezer.com/hc/en-gb/articles/115004522449-Content-Limits)  so maybe not? I dont know, i feel like its wrong to limit it because people using the library expect it to fetch all of it right?